lxd/instance/drivers/driver_qemu.go

```diff
@@ -2617,42 +2617,12 @@ type PyUEFIVar struct {
 	Digest string `json:"digest"`
 }
 
-func callPyUEFIVars(stdin io.Reader, stdout io.Writer, efiVarsPath string) error {
```

Comment:
As we dont do squash merges, the commits in this PR will go into the main branch, and so we like PR commits to tell a logical story of the feature and represent the final version of a PR rather than showing the iteration of the PR during the development process. So I wouldn't expect to see this function added in one commit and then removed in another within the same PR. Please can you rebase this PR to show the final version you want to include. Thanks

Reply:
Fixed, thanks!